IntegrationTests/allocation-counter-tests-framework/template/AtomicCounter/Sources/AtomicCounter/src/atomic-counter.c

```diff
@@ -20,6 +20,9 @@
 */    atomic_fetch_add_explicit(&g_ ## name ## _counter, 1, memory_order_relaxed); /*
 */ } /*
 */ /*
+*/ void add_ ## name ## _counter(int v) { /*
```

Comment:
@ktoso this still uses `int` but it needs to use either `int` or `size_t` I think. We could fix  `MAKE_COUNTER` like `#define MAKE_COUNTER(_type, _name) ...`. The other option would probably be to make everything `ssize_t` or something but that's a bit annoying as we'd get some signed/unsigned conversions.

Reply:
actually making everything `intptr_t` might be the right call. That matches Swift's `Int` which is the type that we propagate to the user in Swift.

Reply:
Is changing int here to `size_t` good enough?

Reply:
@ktoso `size_t` is unsigned

Reply:
Right; summing up though, those to become accepting `size_t` then and we're good...?

Reply:
@ktoso if we have one type for all, it has to be `intptr_t`, right? In the overridden `malloc` function we then need to do the (explicit) cast.

Reply:
Or `ssize_t`?

Reply:
wanted to match `Swift.Int` as closely as possible. Pretty much all platforms that matter have pointer size == word size so `intptr_t` is kinda good. Exceptions are `x32` and Watch Series 4 and maybe WoW64?

Reply:
Sounds like `intptr_t` it is then; I'll check that tomorrow and check why CI failed. 
Sorry for being slow here